# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/IPCTransportProvider.cpp
# Commit: 2502449c2e24
# Full Hash: 2502449c2e24993d495c886c717e7f9546cd9e17
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 432c7e9dc19f -r 2502449c2e24 netwerk/protocol/websocket/IPCTransportProvider.cpp
--- a/netwerk/protocol/websocket/IPCTransportProvider.cpp	Mon Aug 03 10:03:08 2020 +0000
+++ b/netwerk/protocol/websocket/IPCTransportProvider.cpp	Wed Aug 05 15:17:38 2020 +0000
@@ -49,6 +49,12 @@
 }
 
 NS_IMETHODIMP
+TransportProviderParent::OnWebSocketConnectionAvailable(
+    nsIWebSocketConnection* aConnection) {
+  return NS_OK;
+}
+
+NS_IMETHODIMP
 TransportProviderParent::OnUpgradeFailed(nsresult aErrorCode) { return NS_OK; }
 
 void TransportProviderParent::MaybeNotify() {